src/gc-arena/tests/ui/bad_gc_mutate_maro_call.rs

```diff
@@ -0,0 +1,28 @@
+use gc_arena::{Collect, Gc, GcCell};
```

Comment:
Typo in file name: `maro` instead of `macro`

Reply:
Fixed